kadi/commands/command_sets.py

```diff
@@ -100,6 +100,11 @@ def cmd_set_scs107(date=None):
         dict(type="ACISPKT", tlmsid="AA00000000", dur=1.025),
         dict(type="ACISPKT", tlmsid="AA00000000", dur=10.25),
         dict(type="ACISPKT", tlmsid=pow_cmd),
+        dict(type="COMMAND_HW", tlmsid="215PCAOF"),
+        dict(type="COMMAND_HW", tlmsid="224PCAOF"),
+        dict(type="COMMAND_HW", tlmsid="2IMHVOF"),
+        dict(type="COMMAND_HW", tlmsid="2SPHVOF"),
+        dict(type="COMMAND_HW", tlmsid="2S2HVOF"),
```

Comment:
For the hiccup today, HRC and FOT MP settled on using these 8 
```
COMMAND_HW | TLMSID=224PCAOF
COMMAND_HW | TLMSID=2IMHVOF
COMMAND_HW | TLMSID=2SPHVOF
COMMAND_HW | TLMSID=215PCAOF
COMMAND_HW | TLMSID=2S2STHV
COMMAND_HW | TLMSID=2S1STHV
COMMAND_HW | TLMSID=2S2HVOF
COMMAND_HW | TLMSID=2S1HVOF
```
so it sounds like there's a little remaining work figuring out which set makes the most sense.